src/components/Box/Box.tsx

```diff
@@ -10,11 +10,12 @@ import "./index.scss";
 export interface BoxProps {
   children: React.ReactNode;
   title: string;
+  dataQa?: string | null;
```

Comment:
does it really need `null` type here? I noticed that it already has optional `?` type

Reply:
yes. `qa` util in AP returns string | null